lib/Service/Appointments/BookingCalendarWriter.php

```diff
@@ -113,7 +113,7 @@ public function write(AppointmentConfig $config,
 			'CALSCALE' => 'GREGORIAN',
 			'VERSION' => '2.0',
 			'VEVENT' => [
-				'SUMMARY' => $config->getName(),
+				'SUMMARY' => $displayName . ' - ' . $config->getName(),
```

Comment:
**suggestion:** While the order is debatable, in any case this should be localisable (for RTL languages and such).
```suggestion
                                // TRANSLATORS Title for event appoinment, first the attendee name, then the appointment name
				'SUMMARY' => $this->l10n->t('%1$s - %2$s', [$displayName, $config->getName()]),
```

Otherwise it could be something like « $appointmentName with $attendeeName »

Reply:
Much better @tcitworld  :) I went for name first because otherwise in my calendar I would still only see "Really long appointment name... " and wouldn't get to see the actual name, so they would all still appear the same.

Name first means I can at least get: "JOHN SMITH - Really long app...."